### ISSUE TITLE
`comment_with_prefix` should normalize the content

### ISSUE THREAD

**Issue by lmolkova** [open-telemetry/weaver#266]
We can do a better job in semantic conventions wrt trailing dots (we don't 😿 ), but build-tools compensated for it by removing trailing dots at the end of the note or brief. 

`comment_with_prefix` could implement this behavior, which is apparently not trivial to do in jinja.

```jinja 
{%- macro remove_trailing_dots(str) -%}
{%- if str[-1:] == '.' -%}{{ remove_trailing_dots(str[:-1]) }}{%- else -%}{{ str }}{%- endif -%}
{%- endmacro -%}

{%- macro to_docstring(str, prefix) -%}
{{remove_trailing_dots(str | trim(' \n')) | comment_with_prefix(prefix)}}
{%- endmacro %}
```

IT could also be useful to trim any whitespace characters that might or might not be enforced by they yamllint in semantic conventions.

**Comment by lmolkova** [comment]
Also maybe need to escape `\` - need to check if java/c#/any other language needs it, python linters complain about docstring  with unescaped `\`:

<img width="1253" alt="image" src="https://github.com/user-attachments/assets/7886be17-08b0-4f00-94ee-5ad959027022">

**Comment by lquerel** [comment]
See also #291

**Comment by lquerel** [comment]
Implemented in #309
